book-a-reading-room-visit.domain/Booking.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
```

Comment:
please undo all the changes made to the domain classes Booking.cs & OrderDocument.cs